src/main/java/pl/betoncraft/betonquest/commands/QuestCommand.java

```diff
@@ -1553,37 +1632,138 @@ private void handleDeleting(CommandSender sender, String[] args) {
 
 	/**
 	 * Returns a list including all possible options for tab complete of the /betonquest delete command
+	 *
 	 * @param sender
 	 * @param args
 	 * @return
 	 */
 	private List<String> completeDeleting(CommandSender sender, String[] args) {
-		if (args.length == 2) return Arrays.asList("tag", "point", "objective", "entry");
+		if (args.length == 2)
+			return Arrays.asList("tag", "point", "objective", "entry");
 		if (args.length == 3) {
 			switch (args[1].toLowerCase()) {
-				case "tags":
-				case "tag":
-				case "t":
-				case "points":
-				case "point":
-				case "p":
-					return completeId(sender, args, null);
-				case "objectives":
-				case "objective":
-				case "o":
-					return completeId(sender, args, ConfigAccessor.AccessorType.OBJECTIVES);
-				case "journals":
-				case "journal":
-				case "j":
-				case "entries":
-				case "entry":
-				case "e":
-					return completeId(sender, args, ConfigAccessor.AccessorType.JOURNAL);
+			case "tags":
+			case "tag":
+			case "t":
+			case "points":
+			case "point":
+			case "p":
+				return completeId(sender, args, null);
+			case "objectives":
+			case "objective":
+			case "o":
+				return completeId(sender, args, ConfigAccessor.AccessorType.OBJECTIVES);
+			case "journals":
+			case "journal":
+			case "j":
+			case "entries":
+			case "entry":
+			case "e":
+				return completeId(sender, args, ConfigAccessor.AccessorType.JOURNAL);
 			}
 		}
 		return new ArrayList<>();
 	}
 
+	/**
+	 * Creates a npc at the entity/block the player is clicking next
+	 */
+	private void createNPC(CommandSender sender, String[] args) {
+		if (sender instanceof Player) {
```

Comment:
I would put an early return here:

```
if (!(sender instanceof Player)) {
  // send a message
  return;
}
```

We'll have one less level of indentation and the code will be more readable.